README.md

```diff
@@ -127,7 +127,7 @@ Subsets of all available tests can be selected via `--gtest_filter=`.
 ### Coverage
 `./scripts/coverage.sh` will print a summary to the command line and create detailed html reports at ./coverage/index.html
 
-*Supports only clang on MacOS and only gcc on linux*
+*Supports only clang on both MacOS and Linux.*
```

Comment:
Does the coverage script "support" clang? Sounds somewhat confusing to me.

"Requires clang on macOS and Linux"?   (macOS is the official name now)